website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -642,6 +642,17 @@ as they construct keys using it.
     resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
 />
 
+### S3 Inventory Report
+
+S3 Inventory reports are created on S3 buckets and post information about that bucket's contents to another bucket.
+For our purposes, the following options must be checked:
+
+- Size
+- Last modified
+- Storage class
+- ETag
+
```

Comment:
Did you need to check on anything related to the deletes or have any additional configuration to make them show up?

Reply:
That's the "Include all versions" bit.

Reply:
That's actually pretty bad... Shouldn't have tried to work that in. Reverting.

Reply:
Cool, thank you